packages/renderer/src/providers/web-rendering-template-2024.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  IRendererContext,
+  IRendererProvider,
+  RenderDocument,
+} from '@vckit/core-types';
+import handlebars from 'handlebars';
+/**
+ * @public
+ */
+export class WebRenderingTemplate2024 implements IRendererProvider {
+  async renderCredential({
+    template,
+    document,
+    url,
+    digestMultibase,
+    context,
+  }: {
+    template?: string;
+    document?: RenderDocument;
+    url?: string;
+    digestMultibase?: string;
+    context?: IRendererContext;
+  }): Promise<string> {
+    try {
+      let svgTemplate: string | any = '';
```

Comment:
Why the name of the variable is `svgTemplate `? And the type should be only `string`